packages/editor/src/plugins/rows/components/add-row-button-floating.tsx

```diff
@@ -1,56 +1,72 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
 import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
-import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
-import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-context'
 import { cn } from '@serlo/frontend/src/helper/cn'
 import { useRef } from 'react'
 
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
 interface AddRowButtonFloatingProps {
-  onClick: React.MouseEventHandler<HTMLButtonElement>
+  onClick: (event: React.MouseEvent | React.KeyboardEvent) => void
   focused?: boolean
   hide?: boolean
 }
 
 export function AddRowButtonFloating({
-  focused,
   onClick,
+  focused,
   hide,
 }: AddRowButtonFloatingProps) {
   const rowsStrings = useEditorStrings().plugins.rows
   const buttonRef = useRef<HTMLButtonElement>(null)
 
+  if (hide) return null
+
+  const baseStyles = `
+  group z-50
+  opacity-0 transition-opacity duration-300
+  relative mb-6 flex items-center px-8 
+`
+
+  const stateStyles = `
+  text-editor-primary-200 hover:opacity-100 ${focused ? 'opacity-40' : ''}
+  focus-visible:text-editor-primary-200 group-hover:opacity-100
+`
+
+  const interactionStyles = 'cursor-pointer'
+
   return (
-    <div className="absolute bottom-0 z-[1] h-auto w-full translate-y-full">
-      {hide ? null : (
-        <div className="group flex justify-center">
-          <button
-            className={cn(
-              `
-              add-trigger z-20 h-7 w-7
-              rounded-full pt-1 text-center
-              text-gray-600 transition-all hover:cursor-pointer
-              hover:text-editor-primary hover:opacity-100
-              focus-visible:text-editor-primary group-hover:opacity-60
-              `,
-              focused ? 'opacity-60' : 'opacity-0'
-            )}
-            title={rowsStrings.addAnElement}
-            onClick={(event) => {
-              onClick(event)
-
-              // Imperatively calling blur, because inside the Shadow DOM, the
-              // button somehow retains focus which shifts the suggestion menu
-              // to the right!
-              if (buttonRef.current) {
-                buttonRef.current.blur()
-              }
-            }}
-            data-qa="add-new-plugin-row-button"
-            ref={buttonRef}
-          >
-            <FaIcon icon={faCirclePlus} className="text-xl" />
-          </button>
-        </div>
-      )}
+    <div
+      style={{
+        backgroundColor: 'rgba(255, 255, 255, 0.6)',
+      }}
+      tabIndex={0}
+      onKeyDown={(event) => {
+        if (event.key === 'Enter') {
+          onClick(event)
+        }
+      }}
```

Comment:
You could probably get rid of `tabindex` and the custom `onKeyDown` logic if you would use a `button` instead of a `div` here. Also: better a11y for free.
(then you would have to also use `span` inside the button instead of the divs)

Reply:
Thanks, done :heavy_check_mark:

Reply:
> (then you would have to also use span inside the button instead of the divs)

did you see that part?

Reply:
Done :heavy_check_mark: